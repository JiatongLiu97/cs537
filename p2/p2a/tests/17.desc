getnumsyscalls and getnumsyscallsgood test: A few good syscalls and a few bad syscalls
