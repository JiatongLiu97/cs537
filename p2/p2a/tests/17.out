XV6_TEST_OUTPUT 603 201
